CHANGES.rst

```diff
@@ -5,7 +5,8 @@ Changelog
 0.40 (unreleased)
 -----------------
 
-- Nothing changed yet.
+- Added ``-q``/``--quiet`` command line argument. This will reduce the verbosity
+of informational output, e.g. for use in a CI pipeline.
```

Comment:
The second line needs to be indented a bit to make the ReStructuredText parser happy:

```suggestion
  of informational output, e.g. for use in a CI pipeline.
```